R/utils.R

```diff
@@ -842,6 +851,16 @@ wb_dims <- function(..., select = NULL) {
     }
   }
 
+  # Correct for empty data frame?
```

Comment:
Is this a question? Because I don’t know either 😄

Reply:
All good. I wrote this before I understood the problem. added a clearer comment.
 so the fix just makes sure we span at least one colum.

Reply:
Thanks, I only asked, because sometimes I wonder who wrote a certain comment and why